src/params.cpp

```diff
@@ -79,6 +79,11 @@ Params::Params(int argc, char* const argv[])
 			ok = true;
 		}
 
+		if ((strcmp(argv[i], "--stratum-ban-time") == 0) && (i + 1 < argc)) {
+			m_stratumBanTime = strtoull(argv[++i], nullptr, 10);
```

Comment:
One final thought, it would be also nice to add range checking for this value to the `Params::valid()` function. Too low (less than 10 seconds) and too high (more than a day or so) values don't make much sense.